components/contract-components/hooks.ts

```diff
@@ -223,7 +221,7 @@ export function useAllVersions(
 ) {
   const sdk = useSDK();
   return useQuery(
-    ["all-releases", publisherAddress, contractName],
+    ["all-releases-2", publisherAddress, contractName],
```

Comment:
this is real sketch, why?

Reply:
I was testing if it was a cache issue, already reverted by the time you commented, I was so confused :D